cvat-ui/src/utils/opencv-wrapper/tracker-mil.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { TrackerModel, TrackingResult } from './opencv-interfaces';
```

Comment:
> cvat/apps/opencv/static/opencv/js/opencv_4.5.4_dev.js

Is there something wrong with OpenCV 4.5.5?

---

> cvat/apps/opencv/static/opencv/js/opencv_123.js

Probably not used.

Reply:
Thanks! `opencv_123.js` was removed.
There is nothing wrong with OpenCV 4.5.5, this patch just was build with a bit older version.